android/signin/app/src/main/java/com/google/samples/quickstart/signin/SignInActivity.java

```diff
@@ -97,6 +97,13 @@ public void onResult(GoogleSignInResult googleSignInResult) {
         }
     }
 
+    @Override
+    protected void onResume() {
+        super.onResume();
+        hideProgressDialog();//To hide {@link ProgressDialog} which comes into view unnecessarily,
```

Comment:
Thanks for this pull request!  Can we just remove all the comments you added in the lifecycle methods?  I think the code explains itself.